extendr-api/src/robj/mod.rs

```diff
@@ -734,65 +734,125 @@ where
 {
     fn as_typed_slice(&self) -> Option<&'a [T]>
     where
-        Self: 'a,
-    {
-        None
-    }
+        Self: 'a;
 
     fn as_typed_slice_mut(&mut self) -> Option<&'a mut [T]>
     where
-        Self: 'a,
-    {
-        None
+        Self: 'a;
+}
+
+trait SExpAccessor {
+    const XP: u32;
+    type AccessorType;
+    fn acessor(x: SEXP) -> *mut Self::AccessorType;
```

Comment:
It's unusual for a method to be a noun. Is there an appropriate verb you can use here? `access()`?

Reply:
Renamed.